Sources/SwiftRefactor/ExpandEditorPlaceholder.swift

```diff
@@ -78,12 +80,15 @@ import SwiftSyntaxBuilder
 /// ```
 struct ExpandSingleEditorPlaceholder: EditRefactoringProvider {
   struct Context {
-    let indentationWidth: Trivia?
-    let initialIndentation: Trivia
-
-    init(indentationWidth: Trivia? = nil, initialIndentation: Trivia = []) {
-      self.indentationWidth = indentationWidth
-      self.initialIndentation = initialIndentation
+    let closureLiteralFormat: BasicFormat
+    let allowNestedPlaceholders: Bool
```

Comment:
Could you add a short doc comment what `allowNestedPlaceholders` does, ideally with a short example. I think that would be useful in the future.

Reply:
Absolutely; done!